.dependabot/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 1
+update_configs:
+  - package_manager: ruby:bundler
+    directory: /
+    update_schedule: daily
+    allowed_updates:
+      # Security updates - we should always do these
+      - match: { update_type: security }
+      - match: { dependency_name: brakeman }
+      # Internal gems - we should always update these
+      - match: { dependency_name: gds-api-adapters }
+      - match: { dependency_name: gds-sso }
+      - match: { dependency_name: govspeak }
+      - match: { dependency_name: govuk_app_config }
+      - match: { dependency_name: govuk_publishing_components }
+      - match: { dependency_name: govuk_schemas }
+      - match: { dependency_name: govuk_sidekiq }
+      - match: { dependency_name: govuk_test }
+      - match: { dependency_name: rubocop-govuk }
+      - match: { dependency_name: plek }
+      - match: { dependency_name: scss_lint-govuk }
+      # Framework gems - we use these to write lots of code
+      - match: { dependency_name: factory_bot_rails }
+      - match: { dependency_name: jasmine }
+      - match: { dependency_name: rails }
+      - match: { dependency_name: rspec-rails }
```

Comment:
Should we include `rspec` if we're including this? (It's likely they have to be updated in tandem)

Reply:
I don't think we get these at the moment, as many of our repos are set to "Only top-level dependencies". Looks like [`rspec-rails` is pretty strict on the underlying version](https://github.com/rspec/rspec-rails/blob/master/rspec-rails.gemspec#L53), so I think we will still get these implicitly. `factory_bot_rails` isn't quite so up-to-date, but I think we'd get any major changes to the underlying gem(s) through a version bump at the top-level.